kep/219-pod-allocatable-ip-scheduling/README.md

```diff
@@ -0,0 +1,61 @@
+# Pod Allocatable IP<!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Use Cases](#use-cases)
+- [Design Details](#design-details)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+This plugin takes a node's `.spec.PodCIDR` into account in a scheduling cycle. When the density of pods in a node is relatively high, the node might not have sufficient allocatable Pod IPs to allocate.
```

Comment:
For now, the defeault scheduler only takes `.status.Allocatable["pods"]` into consideration when scheduling. Is this KEP's main purpose is to switch to use `min(num of .spec.PodCIDR, .status.Allocatable["pods"])`?

Reply:
Yes, that's right. Any suggestions on the implementation details?

Reply:
I think that's a generic improvement and we can achieve it in the default scheduler. Do you want to pursue it in k/k?

Reply:
Yes, I would like to do it. What should I do to pursue this in k/k? @Huang-Wei

Reply:
You can raise an issue describing the idea, and involve sig-scheduling and sig-node to see if it makes sense. No kep is needed as the change is small. After we get to a consensus, you can start the implementation.